+lib/+costs/total_deaths.m

```diff
@@ -0,0 +1,11 @@
+function [nr_of_deaths] = total_deaths(M,l)
```

Comment:
The `[nr_of_deaths]` here tells matlab that this method **has to** have a result variable to run line 8. 

This is not the case for this function. It is therefore best practice to change `[nr_of_deaths]` to `nr_of_deaths`.

Again, for this use case, it does no harm if you leave it this way....